Dockerfile

```diff
@@ -35,8 +36,21 @@ RUN zypper addrepo https://download.opensuse.org/repositories/isv:SUSE:Edge:Edge
     podman \
     createrepo_c \
     helm hauler \
-    nm-configurator && \
-    zypper clean -a
+    nm-configurator
+
+# Make adjustments for running guestfish and image modifications on aarch64
+# guestfish looks for very specific locations on the filesystem for UEFI firmware
+# and also expects the boot kernel to be a portable executable (PE), not ELF.
+RUN if [[ $(uname -m) == "aarch64" ]]; then \
+	zypper install -y qemu-uefi-aarch64; \
+	mkdir -p /usr/share/edk2/aarch64; \
+	cp /usr/share/qemu/aavmf-aarch64-code.bin /usr/share/edk2/aarch64/QEMU_EFI-pflash.raw; \
+	cp /usr/share/qemu/aavmf-aarch64-vars.bin /usr/share/edk2/aarch64/vars-template-pflash.raw; \
+	mv /boot/vmlinux* /boot/backup-vmlinux; \
+fi
+
+# Clean up the repos to reduce size
+RUN zypper clean -a
```

Comment:
This needs to be done in the same layer as the installation, otherwise it will still be there.